src/components/devices/openwb_evu_kit/counter.vue

```diff
@@ -11,6 +11,7 @@
 				{ value: 0, text: 'MPM3PM' },
 				{ value: 1, text: 'Lovato' },
 				{ value: 2, text: 'SDM630/SDM72D-M' },
+				{ value: 3, text: 'B23' },
```

Comment:
```suggestion
				{ value: 3, text: 'B23' },
				{ value: 1, text: 'Lovato' },
				{ value: 0, text: 'MPM3PM' },
				{ value: 2, text: 'SDM630/SDM72D-M' },
```
Ich bin immer ein Freund davon, Auswahllisten alphabetisch zu sortieren. Dem Anwender sind die dahinter liegenden Werte ja ziemlich egal.